guides/doc-Upgrading_and_Updating/topics/upgrading_capsule_server.adoc

```diff
@@ -91,7 +95,13 @@ Ensure {SmartProxy} has access to `{RepoRHEL8ServerSatelliteMaintenanceProductVe
 +
 [options="nowrap" subs="attributes"]
 ----
-# {foreman-maintain} self-upgrade
+# subscription-manager repos --enable \
+{RepoRHEL8ServerSatelliteMaintenanceProductVersion}
+
+# subscription-manager repos --enable \
+{RepoRHEL8ServerSatelliteCapsuleProductVersion}
+
+# {package-manager} module enable {project-context}-{smart-proxy-context}
```

Comment:
It's hardcoded in `guides/common/modules/proc_configuring-repositories-proxy.adoc` too so I wouldn't bother with attributes:
```suggestion
# {package-manager} module enable satellite-capsule:el8
```

A better way is to actually define an attribute, but for 3.3 I don't insist on it.